src/genie/libs/parser/iosxe/show_ip.py

```diff
@@ -110,5 +110,250 @@ def cli(self, output = None):
             show_output = self.device.execute(self.cli_command)
         else:
             show_output = output
-    
+
         return super().cli(output = show_output)
+
+
+# =======================================
+# Schema for:
+#  * 'show ip bgp vpnv4 vrf {vrf} summary'
+# =======================================
+class ShowIpBgpVpnv4VrfUserSummarySchema(MetaParser):
+    """Schema for show ip bgp vpnv4 vrf {vrf} summary."""
```

Comment:
this parser already exists... Please check if this is what you need and if yes, please decline this PR.
![image](https://user-images.githubusercontent.com/34322875/97721710-2b641180-1aa0-11eb-858f-e68ca1d8d591.png)

example:
```
>>> dev.parse('show ip bgp vpnv4 vrf VRF1 summary')

2020-10-30 11:06:34,313: %UNICON-INFO: +++ R1_xe: executing command 'show ip bgp vpnv4 vrf VRF1 summary' +++
show ip bgp vpnv4 vrf VRF1 summary
BGP router identifier 10.1.1.1, local AS number 65000
BGP table version is 26, main routing table version 26
2 network entries using 512 bytes of memory
2 path entries using 272 bytes of memory
2/2 BGP path/bestpath attribute entries using 608 bytes of memory
2 BGP extended community entries using 500 bytes of memory
0 BGP route-map cache entries using 0 bytes of memory
0 BGP filter-list cache entries using 0 bytes of memory
BGP using 1892 total bytes of memory
BGP activity 8/0 prefixes, 13/5 paths, scan interval 60 secs
2 networks peaked at 18:38:40 Oct 27 2020 UTC (2d20h ago)

Neighbor        V           AS MsgRcvd MsgSent   TblVer  InQ OutQ Up/Down  State/PfxRcd
10.2.2.2        4        65000       0       0        1    0    0 never    Idle
10.3.3.3        4        65000    4106    4516       26    0    0 2d20h           1

R1_xe#
{'bgp_id': 65000, 'vrf': {'VRF1': {'neighbor': {'10.2.2.2': {'address_family': {'vpnv4': {'version': 4, 'as': 65000, 'msg_rcvd': 0, 'msg_sent': 0, 'tbl_ver': 1, 'input_queue': 0, 'output_queue': 0, 'up_down': 'never', 'state_pfxrcd': 'Idle', 'route_identifier': '10.1.1.1', 'local_as': 65000, 'bgp_table_version': 26, 'routing_table_version': 26, 'attribute_entries': '2/2', 'prefixes': {'total_entries': 2, 'memory_usage': 512}, 'path': {'total_entries': 2, 'memory_usage': 272}, 'total_memory': 1892, 'activity_prefixes': '8/0', 'activity_paths': '13/5', 'scan_interval': 60, 'cache_entries': {'route-map': {'total_entries': 0, 'memory_usage': 0}, 'filter-list': {'total_entries': 0, 'memory_usage': 0}}, 'community_entries': {'total_entries': 2, 'memory_usage': 500}}}}, '10.3.3.3': {'address_family': {'vpnv4': {'version': 4, 'as': 65000, 'msg_rcvd': 4106, 'msg_sent': 4516, 'tbl_ver': 26, 'input_queue': 0, 'output_queue': 0, 'up_down': '2d20h', 'state_pfxrcd': '1', 'route_identifier': '10.1.1.1', 'local_as': 65000, 'bgp_table_version': 26, 'routing_table_version': 26, 'attribute_entries': '2/2', 'prefixes': {'total_entries': 2, 'memory_usage': 512}, 'path': {'total_entries': 2, 'memory_usage': 272}, 'total_memory': 1892, 'activity_prefixes': '8/0', 'activity_paths': '13/5', 'scan_interval': 60, 'cache_entries': {'route-map': {'total_entries': 0, 'memory_usage': 0}, 'filter-list': {'total_entries': 0, 'memory_usage': 0}}, 'community_entries': {'total_entries': 2, 'memory_usage': 500}}}}}}}}
>>>